packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -856,6 +856,110 @@ void main() {
     }
   });
 
+  testWidgets('Snackbar labels can be colored (M3 / MaterialColor)',
```

Comment:
Can we rename this to 'Snackbar labels can be coloured as Color (Material 3)' 

Same for the other test, with MaterialStateColor.